boards/imix/src/main.rs

```diff
@@ -172,6 +175,14 @@ pub unsafe fn reset_handler() {
 
     set_pin_primary_functions();
 
+    let power_config = power::ModulePowerConfig {
+        rf233: true,
+        nrf51422: true,
+        sensors: true,
+        trng: true,
+    };
+    power::configure_module_power(power_config);
```

Comment:
Seems like a more natural use of this is to use the `ModulePowerConfig` struct in place of named parameters, so just:

```rust
power::configure_module_power(power::ModulePowerConfig {
    rf233: true,
    nrf51422: true,
    sendosr: true,
    trng: true,
});
```

instead of pulling out the variable.